pkgs/eslint-plugin/docs/rules/no-allow-console.md

```diff
@@ -6,11 +6,11 @@ Examples of **incorrect** code for this rule:
 
 ```js
 // given filename /path/to/test.js
-beforeEach(function() {
+beforeEach(function () {
```

Comment:
Why did this change? did we update this rule?

Reply:
This stylistic change occurred somewhere after 2.0 was released. There was some pushback from the community, but the space camp won and there’s no config to change it.